picli/command/style.py

```diff
@@ -25,21 +25,15 @@ def execute(self):
         """

```

Comment:
See comment above, you've done a good job of simplifying here, but it's not DRY this way... should be low lift to get it into a method on the superclass